incubation.md

```diff
@@ -0,0 +1,129 @@
+# Jupyter Incubation Process
+
+Incubation is a process by which new Jupyter Subprojects can be created and evaluated for
+inclusion in the main Jupyter organization. In this context a Subproject is a GitHub
+repository with a development team that follows the Jupyter governance and contribution model.
+
+The incubation process doesn't apply to all new Subprojects, only those with the following
+characteristics:
```

Comment:
I think it would be even better to just say, "The incubation process may apply when a Subproject has the following characteristics:" - that is even softer than *will* apply, and can be evaluated on a case-by-case basis.